stewajos/team-justice-in-alloc
# adding firebase stuff

to test,
- open app
- check https://console.firebase.google.com/u/3/project/project-alek/firestore/data~2Ftesting~2Ftarget
- make sure count increments
